fuzzbench/run_experiment.py

```diff
@@ -46,8 +46,14 @@ def run_experiment():
 
     while True:
         print('Current status of jobs:')
+        print('\tstarted:\t%d' % queue.started_job_registry.count)
+        print('\tdeferred:\t%d' % queue.deferred_job_registry.count)
+        print('\tfinished:\t%d' % queue.finished_job_registry.count)
+        print('\tscheduled:\t%d' % queue.scheduled_job_registry.count)
```

Comment:
isn't the scheduled_jobs_registry contain jobs that were scheduled to be enqueued at a later time by with enqueue_at() or enqueue_in()?

Reply:
Good question! From offline discussion, this will be confirmed by not running workers and check whether there is only one scheduled job in this registry, saying `base-image`.
I will check this later.

Reply:
`len(queue)` is not `queue.scheduled_job_registry.count`.

> the scheduled_jobs_registry contain jobs that were scheduled to be enqueued at a later time by with enqueue_at() or enqueue_in()

I think yes according to this doc link: https://python-rq.org/docs/scheduling/#scheduling-jobs-for-execution

```
run-experiment_1  | Current status of jobs:
run-experiment_1  |     queued: 1
run-experiment_1  |     started:        0
run-experiment_1  |     deferred:       15
run-experiment_1  |     finished:       0
run-experiment_1  |     scheduled:      0
run-experiment_1  |     failed: 0
run-experiment_1  |   fuzzbench.jobs.build_image : queued       (base-image)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (base-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (base-runner)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (coverage-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (coverage-freetype2-2017-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (afl-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (afl-freetype2-2017-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (afl-freetype2-2017-intermediate-runner)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (coverage-libpng-1.2.56-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (afl-libpng-1.2.56-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (afl-libpng-1.2.56-intermediate-runner)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (libfuzzer-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (libfuzzer-freetype2-2017-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (libfuzzer-freetype2-2017-intermediate-runner)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (libfuzzer-libpng-1.2.56-builder)
run-experiment_1  |   fuzzbench.jobs.build_image : deferred     (libfuzzer-libpng-1.2.56-intermediate-runner)
```

Reply:
So, given that we don't "schedule" jobs, should we remove the  
`print('\tscheduled:\t%d' % queue.scheduled_job_registry.count)`
line?

Reply:
Sounds good.

Reply:
By the way, I think we still have reasons to keep it as it will print all registries' info, no missing, which is complete.
wdyt?

Reply:
i personally don't see a reason the print "scheduled: 0" every few seconds, when we never schedule jobs

Reply:
OK. Done for removal.